docs/src/App.tsx

```diff
@@ -4,7 +4,7 @@ import * as vegaLite from 'vega-lite';
 
 // which results are attempted to load
 // the first is selected automatically
-const NODE_VERSIONS = [23, 22, 21, 20, 19, 18, 16];
+const NODE_VERSIONS = [24, 23, 22, 21, 20, 19, 18, 16];
```

Comment:
Same as with bun, non-benchmarked node versions (16, 18, 19) should not be shown in the app.